AssistEm/assistem-android
# Changed GCM sender ID and send gcmid every time a user logs in

to test:
make a new patient
make a new caretaker
ping from patient, should receive push notification
